src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -14,6 +14,20 @@ export type HierarchyWithFields<T extends {fields?: string[]}> = Omit<T, 'fields
     fields: string[];
 };
 
+interface Cached<T> {
+    item: T;
+    lastUpdated: Date;
```

Comment:
Just noticed this, should it be `number` instead? :thinking: 

Validated using the browser console: `JSON.parse("{ \"lastUpdated\": 1599599728072 }").lastUpdated.getDate()` this fails!

Reply:
I guess on the interface it should be number, but the intention is that you do `new Date(lastUpdated)`
where should that happen?

Reply:
Yeah

Reply:
I assumed the conversion would be made automagically. 

I'll change it for a `number` as it's what other parts of the codebase use for dates (https://github.com/coveo/platform-client/blob/master/src/resources/Extensions/ExtensionsInterfaces.ts#L4, https://github.com/coveo/platform-client/blob/master/src/resources/CrawlingModule/CrawlingModuleInterfaces.ts#L8, https://github.com/coveo/platform-client/blob/master/src/resources/Organizations/OrganizationInterfaces.ts#L11)